cmd/gcp-controller-manager/BUILD

```diff
@@ -50,6 +50,7 @@ go_library(
         "//pkg/tpmattest",
         "//providers/gce",
         "//vendor/cloud.google.com/go/compute/metadata",
+        "//vendor/github.com/cenkalti/backoff/v4:backoff",
```

Comment:
I dont' think we are using this anymore. Please remove.

Reply:
done.